packages/office-ui-fabric-react/src/components/ProgressIndicator/ProgressIndicator.test.tsx

```diff
@@ -5,6 +5,12 @@ import { ProgressIndicator } from './ProgressIndicator';
 
 describe('ProgressIndicator', () => {
   it('renders ProgressIndicator correctly', () => {
+    const component = renderer.create(<ProgressIndicator percentComplete={ 0.75 } />);
+    let tree = component.toJSON();
```

Comment:
This looks like it could be a const as well unless tree is mutated further in the process chain.